### PR TITLE
FIX: BEND-71 Segmentation route controller and action code

### PR DESCRIPTION
**Pull request by Kehomaxd**
we structure the actions, controllers and routes, segmented by works, educations, photos and languages.
Also we fixed dates and others errors of sprint 3

### CODE REVIEW

**Review by OvidioSantoro — DISMISSED** [review]
I think everything's ok!

**Review by davidcode10 — DISMISSED** [review]
Approved

File: src/http/controller/User.languages.ts [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+import {UserLanguagesActions} from '../actions/User.languages';
+import {Languages} from '../../models/domain/domain.languages';
+
+// * Environments
+const _microservice = 'User';
+const _version = 'v1.0.0';
+const _date = Date.now();
+
+/**
+ * !segmentation languages controller
```
Comment (octapf) [inline src/http/controller/User.languages.ts:19]: @Kehomaxd spaces and capital letter missing

File: src/http/controller/User.photos.ts [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+import {PhotoDTOManager} from '../../models/types/gen/dto/dto.photo';
+import { UserActionsPhotos } from '../actions/User.photos';
+
+// * Environments
+const _microservice = 'User';
+const _version = 'v1.0.0';
+const _date = Date.now();
+
+/**
+ * !segmentation routes controller and actions code
```
Comment (octapf) [inline src/http/controller/User.photos.ts:19]: @davidcode10 change this description so matches the others

File: src/http/controller/User.photos.ts [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+ * * davidcode10 - 2022/02/10
+ */
+export module UserControllerPhotos {
```
Comment (octapf) [inline src/http/controller/User.photos.ts:22]: @davidcode10 Should be UserPhotosController

File: src/routes/User.educations.ts [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+import {JWTMiddleware} from '../http/middleware/middleware.jwt';
+import {UserEducationController} from '../http/controller/User.educations';
+import {UserEndpoints} from '../http/common/Base/static/Base.UserEndpoints';
+import {PermissionRoles} from '../models/types/gen/gen.permissions';
+
+// * Environment
+const _resource = 'USER';
+
+/**
+ * !segmentation education routes
```
Comment (octapf) [inline src/routes/User.educations.ts:12]: @Kehomaxd change file name and this description to singular

**Review by octapf — CHANGES_REQUESTED** [review]

**Review by OvidioSantoro — APPROVED** [review]

**Review by octapf — APPROVED** [review]

**Review by PatyVilas — APPROVED** [review]

**Review by davidcode10 — APPROVED** [review]

**Pull request merged**